docs/data/migration/migration-data-grid-v7/migration-data-grid-v7.md

```diff
@@ -38,6 +38,7 @@ Below are described the steps you need to make to migrate from v7 to v8.
 - The default value of the `rowSelectionPropagation` prop has been changed to `{ parents: true, descendants: true }` which means that the selection will be propagated to the parents and descendants by default.
   To revert to the previous behavior, pass `rowSelectionPropagation={{ parents: false, descendants: false }}`.
 - The prop `indeterminateCheckboxAction` has been removed. Clicking on an indeterminate checkbox "selects" the unselected descendants.
+- The "Select all" checkbox would now be checked when all the selectable rows are selected, even when some rows are not selected because of the `isRowSelectable` prop.
```

Comment:
```suggestion
- The "Select all" checkbox would now be checked when all the selectable rows are selected, ignoring rows that are not selectable because of the `isRowSelectable` prop.
```

Reply:
Definitely better wording, thanks!